src/pages/Companies/index.tsx

```diff
@@ -15,6 +15,8 @@ import InfoCard from 'components/layout/InfoCard';
 import Banner, { BannerButton } from 'components/layout/Banner';
 import CompaniesForm from 'pages/Companies/components/CompaniesForm';
 import Container from 'components/layout/Container';
+import { UserStudy } from '../../types/Enums';
+import { getUserStudyLong } from '../../utils';
```

Comment:
Bruk absolute import :)

```suggestion
import { UserStudy } from 'types/Enums';
import { getUserStudyLong } from 'utils';
```